addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.NZD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.JPY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.CNY" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.MYR" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.SGD" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.GBP" model="res.currency">
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.lang_en" model="res.lang">
```

Comment:
The definition of the language is actually [using the locales](https://github.com/odoo/odoo/blob/8c4f17fd8be08a4989e438a52666b499a3571b63/openerp/addons/base/res/res_lang.py#L52) installed on the machine. If the locales are not present (e.g. I am loading Australian on an American server), the fallback will be done on the default of the server. So on my machine I have the nice `%d/%m/%Y` dates format but on odoo server, the ~~illogical~~ american `%m/%d/%Y` format by default.
So Odoo is conforming to what the OS tells him is the right format.